### PR TITLE
#470 fix

### PR DESCRIPTION
**Pull request by brijeshnec**

### CODE REVIEW

File: Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java [hunk 1]
```diff
@@ -365,14 +374,15 @@ public Object compact(Context activeCtx, String activeProperty, Object element,
...
 						}
-					} else if (isLanguageProperty) {
-						if (expandedProperty.equals(NGSIConstants.NGSI_LD_HAS_LANGUAGE_MAP)) {
-							return compact(activeCtx, activeProperty, expandedValue, compactArrays, endPoint, null,
-									null);
-						} else {
-							continue;
-						}
 					}
+//					else if (isLanguageProperty) {
```
Comment (amitrnec) [inline Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java:378]: Remove unused code..

File: Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java [hunk 2]
```diff
@@ -259,18 +260,21 @@ public Object compact(Context activeCtx, String activeProperty, Object element,
...
-						case NGSIConstants.NGSI_LD_GEOPROPERTY:
-							isGeoProperty = true;
-							break;
-						case NGSIConstants.NGSI_LD_LANGPROPERTY:
-							isLanguageProperty = true;
-							break;
-						case NGSIConstants.NGSI_LD_RELATIONSHIP:
-							isRelationship = true;
-							break;
+							case NGSIConstants.NGSI_LD_GEOPROPERTY:
```
Comment (amitrnec) [inline Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java:266]: Rewrite same code ??

Comment (brijeshnec) [reply Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java:266]: fixed indentation

File: Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java (hunk 1 above)
Comment (brijeshnec) [reply Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java:378]: Left for future purposes in case something goes wrong

**Review by amitrnec — APPROVED** [review]

**Pull request merged**